app.py

```diff
@@ -0,0 +1,94 @@
+import tweepy
+import re
+import json
+import time
+
+
+class AmexOfferBot:
+    
+    FILENAME_CONFIGURATION = "config.json"
+    FILENAME_HASHTAG_MEMORY = "hashtag_memory.json"
+
+    def __init__(self):
+        self.configuration = self.loadConfiguration()
+        self.tweeted_hashtags = self.loadHashtagMemory()
+        self.api = self.authenticateApi()
+
+    def loadConfiguration(self):
```

Comment:
Feeback:

```
Python usually likes method names like this_is_my_method rather than camel case thisIsMyMethod
```

(wtf, I thought I saw it the other way around :D )